Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/hyphenate-limit-chars/index.md

Reviewer: Probably outside the scope of your PR, but flagging it here: `auto` needs to be part of the definition list. `number` can be a single entry that could explain the meaning of one-, two-, three-value cases

hyphenate-limit-chars: unset;
```

### Values

> This property is specified as one to three values, with each being a `<number>` or the keyword `auto`:

Reviewer: ```suggestion
This property is specified as one to three values from the following list:
```
